plugins/search/providers/elastic_search/lib/api/types/KalturaESearchParams.php

```diff
@@ -46,6 +52,11 @@ public function toObject($object_to_fill = null, $props_to_skip = array())
 			$object_to_fill = new ESearchParams();
 		}
 
+		if ($this->objectId && $this->objectIds)
```

Comment:
```suggestion
		if (isset($this->objectId) && isset($this->objectIds))
```
Better to use isset() because it should throw an exception on an empty string as well